relay-cabi/src/processing.rs

```diff
@@ -80,38 +200,37 @@ pub unsafe extern "C" fn relay_valid_platforms(size_out: *mut usize) -> *const R
     platforms.as_ptr()
 }
 
-/// Creates a new normalization processor.
+/// Creates a new normalization config.
 #[no_mangle]
 #[relay_ffi::catch_unwind]
 pub unsafe extern "C" fn relay_store_normalizer_new(
     config: *const RelayStr,
     _geoip_lookup: *const RelayGeoIpLookup,
 ) -> *mut RelayStoreNormalizer {
-    let config: StoreConfig = serde_json::from_str((*config).as_str())?;
-    let normalizer = StoreProcessor::new(config);
-    Box::into_raw(Box::new(normalizer)) as *mut RelayStoreNormalizer
+    let config: StoreNormalizer = serde_json::from_str((*config).as_str())?;
```

Comment:
```suggestion
    let normalizer: StoreNormalizer = serde_json::from_str((*config).as_str())?;
```

Reply:
same for functions below

Reply:
Done in [59c0c95](https://github.com/getsentry/relay/pull/3097/commits/59c0c950929b63e64998e8ee817a21e7812b8dba).